Make Snowflake Hook conform to semantics of DBApi (#28006)
mistercrunch/Airflow: airflow/providers/snowflake/CHANGELOG.rst

4.0.1
.....

.. warning::

>     This version is yanked, as it contained problems when interacting with common.sql provider. Pleas install
>     the versions that were released after.

Reviewer: ```suggestion
    This version is yanked, as it contained problems when interacting with common.sql provider. Please install
    a version released afterwards.
```

4.0.0
.....

.. warning::

>     This version is yanked, as it contained problems when interacting with common.sql provider. Pleas install
>     the versions that were released after.

Reviewer: ```suggestion
    This version is yanked, as it contained problems when interacting with common.sql provider. Please install
    a version released afterwards.
```
